build.gradle

```diff
@@ -66,7 +66,7 @@ ext {
     jSoupVersion = '1.11.3'
     wordpressUtilsVersion = 'trunk-1ed207c03d2242b6fc3d74f9e388e9163cbc82a6'
     espressoVersion = '3.0.1'
-    commonTargetSdkVersion = 30
+    commonTargetSdkVersion = 31
```

Comment:
👋 @AjeshRPai !

Maybe we can take this opportunity, in this PR, or, most probably, in a separate PR afterwards (as to not block this one) to do the following as well:

- Rename the `commonTargetSdkVersion` and `commonMinSdkVersion` to `targetSdkVersion` and `minSdkVersion` respectfully to much our usual such naming convension. PS: The 
[about-automattic-android](https://github.com/Automattic/about-automattic-android) also has it's own such naming convention too, which utilizes underscores instead of a camel-case naming pattern, maybe we should consider updating that at some point too.
- Try to extract and then update the `compileSdkVersion` to `31` on all modules (libs and app included), I am seeing it being `28` and I wonder if we can easily update it to `31`, which will give us another additional safety net when testing such upgrades in the future.

Wdyt? 🤔

Reply:
@ParaskP7 
I agree. I wish I had catched that change earlier. I will add it to my to-do and take it up in a separate PR since I don't want to block this PR. WDYT?. 🤔 
Thanks for pointing it out.